### DISCUSSION TITLE
Edge browser doesn't support amPmAriaLabel prop?

### DISCUSSION THREAD

**Discussion by anonymous21155** [wojtekmaj/react-time-picker#321] [Q&A]
My default browser is Edge. I used amPmAriaLabel prop in time picker component, but the AM/PM select input is not appearing. then I tried it on Chrome browser and its visible. Is this a known limitation?

**Comment by wojtekmaj** [comment]
`amPmAriaLabel` does not make AM/PM select magically appear, but simply adds aria-label to AM/PM select _if it's there_. Whether or not it's there depends on (from lowest to highest priority):

* browser locale settings
* OR `locale` prop if you've passed it
* OR `format` prop if you've passed it
